docs/UserGuide.md

```diff
@@ -36,213 +148,261 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 
 6. Refer to the [Features](#features) below for details of each command.
 
---------------------------------------------------------------------------------------------------------------------
+### 3.2 Graphical User Interface Layout:
+<a id="graphical-user-interface-layout"></a>
 
-## Features
+#### 3.2.1 User Interface Overview:
+<a id="user-interface-overview"></a>
 
-<box type="info" seamless>
+TO BE DONE
 
-**Notes about the command format:**<br>
+#### 3.2.2 Additional UI Components:
+<a id="additional-ui-components"></a>
 
-* Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
-  e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`.
+TO BE DONE
 
-* Items in square brackets are optional.<br>
-  e.g `n/NAME [t/TAG]` can be used as `n/John Doe t/friend` or as `n/John Doe`.
+### 3.3 How to use TrackMate commands:
+<a id="how-to-use-track-mate-commands"></a>
 
-* Items with `…`​ after them can be used multiple times including zero times.<br>
-  e.g. `[t/TAG]…​` can be used as ` ` (i.e. 0 times), `t/friend`, `t/friend t/family` etc.
+TrackMate operates primarily through text-based commands. Before we explore the specific commands in detail in the 
+Commands section, let’s familiarize ourselves with the basic components and format of a command.
 
-* Parameters can be in any order.<br>
-  e.g. if the command specifies `n/NAME p/PHONE_NUMBER`, `p/PHONE_NUMBER n/NAME` is also acceptable.
+#### 3.3.1 Parameter Prefixes:
```

Comment:
I like how you've presented the parameter prefix! It makes it easy to reference them!